scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -273,6 +273,24 @@ public SchedulerDBManager(Configuration configuration, boolean drop) {
         return new Page<>(lJobs, totalNbJobs);
     }
 
+    public List<JobInfo> getJobs(final List<String> jobIds) {
+        List<Long> longJobIds = jobIds.stream().map(id -> Long.parseLong(id)).collect(Collectors.toList());
```

Comment:
You could add `.distinct()` to evict duplicate ids.

Reply:
makes sense, indeed

Reply:
I'm afraid it might affect performance (see the distinct() documentation ) and currently it doesn't really matter as the frontend will handle this very well

Reply:
To avoid using distinct you could also work with a Set.
`Set<Long> longJobIds = jobIds.stream().map(id -> Long.parseLong(id)).collect(Collectors.toSet());`

But if you are confident enough with front-end dealing with duplicates it works with me